contrib/junit.tpl

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" ?>
+<testsuites>
+{{ range . }}
+	{{- $failures := 0 }}
+    {{- range .Vulnerabilities -}}
+	{{- $failures = sum $failures 2 -}}
```

Comment:
I'm just curious why 2 is added. I scanned alpine:3.10.2 and detected 5 vulnerabilities, but the XML file said 10 failures.